scripts/generate-paas-manifest.py

```diff
@@ -31,6 +33,20 @@ def paas_manifest(environment, app, vars_file, out_file):
 
     variables = merge_dicts(variables, variables[app])
 
+    # read variables from command line
+    # these can be key-value pairs, or
+    # the name of an environment variable
```

Comment:
Maybe this could be mentioned in the actual `help` string?

Reply:
Done